step00-start/build.gradle

```diff
@@ -41,12 +41,12 @@ android {
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.appcompat:appcompat:1.1.0-alpha01'
-    implementation 'androidx.recyclerview:recyclerview:1.1.0-alpha01'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-alpha3'
-    implementation 'androidx.lifecycle:lifecycle-extensions:2.1.0-alpha01'
-    kapt "androidx.lifecycle:lifecycle-compiler:2.1.0-alpha01"
-    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
-    implementation 'androidx.core:core:1.1.0-alpha03'
+    implementation project(':shared')
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+    implementation "androidx.appcompat:appcompat:$appCompatVersion"
+    implementation "androidx.recyclerview:recyclerview:$recyclerViewVersion"
```

Comment:
Do you want these transitively from shared? Otherwise change shared's dependencies to api instead of implementation

Reply:
It's the other way around :D `implementation` is non-transitive and `api` is transitive

Reply:
My bad, even I get confused :)